site-src/guides/rollout.md

```diff
@@ -0,0 +1,197 @@
+
+
+```yaml
+apiVersion: inference.networking.x-k8s.io/v1alpha2
+kind: InferenceModel
+metadata:
+  name: food-review
+spec:
+  modelName: food-review
+  criticality: Standard
+  poolRef:
+    name: vllm-llama3-8b-instruct
+  targetModels:
+  - name: food-review-1
+    weight: 90
+  - name: food-review-2
+    weight: 10
+```
+
+The above configuration means one in every ten requests should be sent to the new version. Try it out:
+
+1. Get the gateway IP:
+```bash
+IP=$(kubectl get gateway/inference-gateway -o jsonpath='{.status.addresses[0].value}'); PORT=80
+```
+
+2. Send a few requests as follows:
+```bash
+curl -i ${IP}:${PORT}/v1/completions -H 'Content-Type: application/json' -d '{
+"model": "food-review",
+"prompt": "Write as if you were a critic: San Francisco",
+"max_tokens": 100,
+"temperature": 0
+}'
+```
+
+### Finish the rollout
+
+
+Modify the InferenceModel to direct 100% of the traffic to the latest version of the adapter.
+
+```yaml
+apiVersion: inference.networking.x-k8s.io/v1alpha2
+kind: InferenceModel
+metadata:
+  name: food-review
+spec:
+  modelName: food-review
+  criticality: Standard
+  poolRef:
+    name: vllm-llama3-8b-instruct
+  targetModels:
+  - name: food-review-2
+    weight: 100
+```
+
+Unload the older versions from the servers by updating the LoRA syncer ConfigMap to list the older version under the `ensureNotExist` list:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: vllm-llama3-8b-instruct-adapters
+data:
+  configmap.yaml: |
+    vLLMLoRAConfig:
+      name: vllm-llama3-8b-instruct-adapters
+      port: 8000
+      defaultBaseModel: meta-llama/Llama-3.1-8B-Instruct
+      ensureExist:
+        models:
+        - id: food-review-2
+          source: Kawon/llama3.1-food-finetune_v14_r8
+      ensureNotExist:
+        models:
+        - id: food-review-1
+          source: Kawon/llama3.1-food-finetune_v14_r8
+```
+
+With this, all requests should be served by the new adapter version.
+
+## **node-update-rollout**
+Node update roll outs safely migrate inference workloads to new node hardware or accelerator configurations. 
+This process happens in a controlled manner without interrupting model service. 
+Use node update roll outs to minimize service disruption during hardware upgrades, driver updates, or security issue resolution.
+
+1.  **Create a new `InferencePool`**: deploy an `InferencePool` configured with the
+    updated node or hardware specifications.
+1.  **Split traffic using an `HTTPRoute`**: configure an `HTTPRoute` to distribute
+    traffic between the existing and new `InferencePool` resources. Use the `weight`
+    field in `backendRefs` to manage the traffic percentage directed to the new
+    nodes.
+1.  **Maintain a consistent `InferenceModel`**: retain the existing
+    `InferenceModel` configuration to ensure uniform model behavior across both
+    node configurations.
+1.  **Retain original resources**: keep the original `InferencePool` and nodes
```

Comment:
Can you update the steps similar to the LoRA rollout guide above, and provide example yamls for each step, such as:
1. Prepare the new InferencePool and InferenceModels, and EPP
1. Apply traffic split
1. Verify 
1. Finish the rollout 
1. Delete old InferencePool, InferenceModels and EPP